forui/CHANGELOG.md

```diff
@@ -79,6 +78,17 @@ The traversal-edge behavior of `FPopover` and Forui widgets that depend on it ha
 * **Breaking** Change `focusNode` from `FocusNode` to `FocusScopeNode` in `FSelectMenuTile`.
 
 
+### `FHeader`
+
+Several minor tweaks have been made to `FHeader` to improve usability and consistency.
+
+* Change `FHeader(title: ...)` to be optional.
+* Change `FHeader.nested(title: ...)` to be optional.
+* **Breaking** Change `FHeader(actions: ...)` to `FHeader(suffixes: ...)`.
+* **Breaking** Change `FHeader(prefixActions: ...)` to `FHeader(prefixes: ...)`.
+* **Breaking** Change `FHeader(suffixActions: ...)` to `FHeader(suffixes: ...)`.
```

Comment:
Are the website docs updated? I'm confused as well, I don't see any changes to the header.